README.rst

```diff
@@ -1,6 +1,9 @@
 Toil is a scalable, efficient, cross-platform pipeline management system, written entirely in
 Python, and designed around the principles of functional programming. Full documentation can be found at `Read the Docs`_.
 
+Native system requirements: `python2.7-dev`_, make_, gcc_, `libffi-dev`_,
 .. _Read the Docs: http://toil.readthedocs.org/en/latest/
-
-
+.. _python2.7-dev: http://packages.ubuntu.com/search?keywords=python2.7-dev
```

Comment:
This was the most official documentation for the python-dev package I could find, but it still isn't very informative.